### DISCUSSION TITLE
The command php artisan test isn't caught by xdebug

### DISCUSSION THREAD

**Discussion by Romuloalmeida** [laravel/framework#32851] [General]
Hi guys,

I'm struggling with the following behavior:

If I execute `php ./vendor/bin/phpunit` Xdebug will caught any breakpoint I set in the Visual Studio Code.

Otherwise, If I execute `php artisan test` breakpoints are bypassed.

I'm using:
PHP 7.3.9.
Xdebug v2.9.0.
Laravel/framework: 7.0.
OS: Mac OS Catalina
Web Server: Mamp PRO

Thank you in advanced.

**Comment by denisdulici** [comment]
Try to add the following lines in your `php.ini` file:

```
xdebug.remote_autostart=1
xdebug.remote_autorestart=1
```
